modules/references/client/components/ReferencesNew.component.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as references from './references.api';
+import Navigation from './Navigation';
+import Interaction from './Interaction';
+import Recommend from './Recommend';
```

Comment:
These component names feel kinda generic if we imagine that in future there will be also "read references" components under this section. I'd love to keep things in separate subfolders for clarity and if that isn't possible, maybe prefix them somehow, maybe just "NewReference"?